src/mono/wasm/build/WasmApp.targets

```diff
@@ -154,7 +154,19 @@
     <WasmStripILAfterAOT Condition="'$(WasmStripILAfterAOT)' == ''">false</WasmStripILAfterAOT>
 
     <WasmRuntimeAssetsLocation Condition="'$(WasmRuntimeAssetsLocation)' == ''">_framework</WasmRuntimeAssetsLocation>
-    <BlazorCacheBootResources Condition="'$(BlazorCacheBootResources)' == ''">true</BlazorCacheBootResources>
+
+    <!-- Runtime feature defaults to trim unnecessary code -->
+    <InvariantTimezone Condition="'$(InvariantTimezone)' == ''">false</InvariantTimezone>
+    <HybridGlobalization Condition="'$(HybridGlobalization)' == ''">false</HybridGlobalization>
+    <InvariantGlobalization Condition="'$(InvariantGlobalization)' == ''">false</InvariantGlobalization>
+    <EventSourceSupport Condition="'$(EventSourceSupport)' == ''">false</EventSourceSupport>
+    <UseSystemResourceKeys Condition="'$(UseSystemResourceKeys)' == ''">true</UseSystemResourceKeys>
+    <EnableUnsafeUTF7Encoding Condition="'$(EnableUnsafeUTF7Encoding)' == ''">false</EnableUnsafeUTF7Encoding>
+    <HttpActivityPropagationSupport Condition="'$(HttpActivityPropagationSupport)' == ''">false</HttpActivityPropagationSupport>
+    <NullabilityInfoContextSupport Condition="'$(NullabilityInfoContextSupport)' == ''">false</NullabilityInfoContextSupport>
+    <_AggressiveAttributeTrimming Condition="'$(_AggressiveAttributeTrimming)' == ''">true</_AggressiveAttributeTrimming>
+    <DebuggerSupport Condition="'$(DebuggerSupport)' == '' and '$(Configuration)' != 'Debug'">false</DebuggerSupport>
+    <WasmFingerprintDotnetJs Condition="'$(WasmFingerprintDotnetJs)' == ''">false</WasmFingerprintDotnetJs>
```

Comment:
You mean `WasmFingerprintDotnetJs` only, right? `NullabilityInfoContextSupport`, `_AggressiveAttributeTrimming` and `DebuggerSupport` are used to define feature switches and have influence on how we trim the app. Removing trimming and nullability setup has negative effect on the size. Removing debugger support setup does not influence the size but only today, in the future it might make a difference. For reference, feature switches that by default the linker gets with this PR:
```
--feature Microsoft.Extensions.DependencyInjection.VerifyOpenGenericServiceTrimmability true
--feature System.AggressiveAttributeTrimming true
--feature System.ComponentModel.TypeConverter.EnableUnsafeBinaryFormatterInDesigntimeLicenseContextSerialization false
--feature System.Diagnostics.Debugger.IsSupported false
--feature System.Diagnostics.Tracing.EventSource.IsSupported false
--feature System.Globalization.Invariant false
--feature System.Globalization.Hybrid false
--feature System.Net.Http.EnableActivityPropagation false
--feature System.Reflection.Metadata.MetadataUpdater.IsSupported false
--feature System.Reflection.NullabilityInfoContext.IsSupported false
--feature System.Resources.ResourceManager.AllowCustomResourceTypes false
--feature System.Resources.UseSystemResourceKeys true
--feature System.Runtime.InteropServices.BuiltInComInterop.IsSupported false
--feature System.Runtime.InteropServices.EnableConsumingManagedCodeFromNativeHosting false
--feature System.Runtime.InteropServices.EnableCppCLIHostActivation false
--feature System.Runtime.InteropServices.Marshalling.EnableGeneratedComInterfaceComImportInterop false
--feature System.Runtime.Serialization.EnableUnsafeBinaryFormatterSerialization false
--feature System.StartupHookProvider.IsSupported false
--feature System.Text.Encoding.EnableUnsafeUTF7Encoding false
--feature System.Text.Json.JsonSerializer.IsReflectionEnabledByDefault true
--feature System.Threading.Thread.EnableAutoreleasePool false
```